src/js/components/DataTable/DataTable.js

```diff
@@ -70,6 +70,17 @@ const DataTable = ({
   step = 50,
   ...rest
 }) => {
+  // Pulling out the data object for the selected column
+  const gdtSelected = columnsProp.find(
+    column => column.property === 'gdt-selected',
```

Comment:
This look like "gdt-selected" becomes a special property name?

Reply:
Yea, that's a reserved name for the selected column. Currently, the 'selected' column doesn't have an entry on the `columns` prop, this PR is enabling styling for this column under this reserved name.